### DISCUSSION TITLE
Oracle Triggers

### DISCUSSION THREAD

**Discussion by EurekaJiang** [dbeaver/dbeaver#17910] [Q&A]
in Version 22.2.1, take schema 'HR' for instance, use the user 'HR' login in, I can see the tiggers under table 'EMPLYEES', but cannot see them under the folder "Table Tiggers". If there anyone know the reason?

**Comment by LonwoLonwo** [comment]
Hello @AndFiv 

Unfortunately, I can't reproduce it.

![2022-10-13 14_18_44-DBeaver Ultimate 22 2 2 - newtable](https://user-images.githubusercontent.com/45152336/195583058-268b7649-fb73-4c56-b553-8deac9d00943.png)

You can check [Query Manager ](https://github.com/dbeaver/dbeaver/wiki/Query-Manager)(with enabled filters) and find the query for table triggers reading to check its result.

**Reply by EurekaJiang** [reply]
It's OK in English version on mac OS
![(35ZFE}U 8X$V5H9EF(XQMH](https://user-images.githubusercontent.com/41953750/195744736-42612318-8d9c-416d-8ccd-fa6060307043.jpg)

but on Windows 10 Chinese version
![image](https://user-images.githubusercontent.com/41953750/195744095-4e18f045-068f-490b-8c75-adbe15bf3e06.png)

I reinstall the dbeaver, the triggers can be seen now 
![image](https://user-images.githubusercontent.com/41953750/195777826-93dce5ef-53b1-4b0b-b869-e2947689dba8.png)

**Reply by LonwoLonwo** [reply]
Ok... So, is the problem solved?
